vueapp/src/components/Game.vue

```diff
@@ -9,27 +9,31 @@
       <div class="column is-11">
         <div class="box is-full-card">
             <!--div class="box is-card-image is-loading"-->
-            <div class="button is-card-image is-loading">You should never see this text ever ever ever.
-                <div class="game-image"><img src="franeheader.jpg"></div>
+            <div class="button is-card-image is-loading" id="card-loader">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+                &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
As a placeholder I think these are fine but longer term we can do this more cleanly in CSS

Reply:
Yes 👍